### DISCUSSION TITLE
Github Campuss Expert verified the campuss expert github but it is pending and rejected.

### DISCUSSION THREAD

**Discussion by kreeksec** [community/community#141103] [GitHub Education]
### Hi everyone,

a few days ago I verified that I was a lecturer / teacher at several campuses in my current country, then I verified on github campuss expert. but it was rejected, can you help me for verifying.

**Comment by WonDKim** [comment]
### There could be several reasons why your GitHub Campus Expert application or verification was rejected or is still pending:
 1. Eligibility Requirements Not Fully Met: The program has specific requirements, including active community involvement, leadership skills, and a good understanding of GitHub. Review the criteria and make sure all the boxes are ticked.
 1. Insufficient Proof of Community Engagement: One of the most important criteria is your contribution to the campus or open-source communities. If GitHub doesn't see enough evidence of this in your application, it could delay or lead to rejection.
 1. Application Lacks Detail: Your application should clearly detail your involvement in organizing events, leading initiatives, and contributing to the community. If the information is vague or missing, this could be the reason for rejection.
 1. Supporting Documentation Missing: You may need to include links to GitHub repositories, past events, or communities you’ve contributed to or led. Without sufficient evidence, your application might not be strong enough.
 1. Pending Review Process: If your application is still pending, it could simply be due to the high volume of applicants, or that GitHub’s team is still reviewing it. Keep an eye on your email for further updates.

**Reply by kreeksec** [reply]
wow, very fast respond thank you very much sir. today i still read the privacy and steps how to join campuss experts.
